openlibrary/catalog/edition_merge/templates/index.html

```diff
@@ -47,8 +47,9 @@ <h1>Merge duplicate editions</h1>
 {% if unmerge_count == 0 %}
 <td style="background: green; color: white">
 {% else %}
-<td>
+</td>
```

Comment:
This is within an if. Line 50 should not be a close `</td>`.

The code is saying... either the html will be what is shown on L48, else, it will be line 50.

Same with L52 below (unnecessary insertion)

Reply:
okay i will change it